oblib/taxonomy_loader.py

```diff
@@ -0,0 +1,529 @@
+# Copyright 2018 SunSpec Alliance
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Handles Loading of Orange Button Taxonomy.  No external functionality exposed."""
+
+import xml.sax
+import os
+import sys
+
+from oblib import constants, util, taxonomy
```

Comment:
perhaps `from oblib.taxonomy import ConceptDetails` etc. to only  import the classes that will be used.  Might be cleaner.